multi-node/aws/pkg/config/config.go

```diff
@@ -300,25 +328,25 @@ func (cfg Cluster) valid() error {
 		)
 	}
 
-	podNetIP, podNet, err := net.ParseCIDR(cfg.PodCIDR)
+	_, podNet, err := net.ParseCIDR(cfg.PodCIDR)
 	if err != nil {
 		return fmt.Errorf("invalid podCIDR: %v", err)
 	}
-	if vpcNet.Contains(podNetIP) {
+	if vpcNet.Contains(podNet.IP) {
```

Comment:
This seems redundant with https://github.com/coreos/coreos-kubernetes/pull/346/files#diff-e5ad8b828a5de3fe2229bee5c64ab2a0R346